examples/2d/mesh2d_vertex_color_texture.rs

```diff
@@ -29,11 +32,28 @@ fn setup(
     ];
     // Insert the vertex colors as an attribute
     mesh.insert_attribute(Mesh::ATTRIBUTE_COLOR, vertex_colors);
-    // Spawn
+
+    let mesh_handle: Mesh2dHandle = meshes.add(mesh).into();
+
+    // Spawn camera
     commands.spawn_bundle(Camera2dBundle::default());
+
+    // Spawn the quad with vertex colors
+    commands.spawn_bundle(MaterialMesh2dBundle {
+        mesh: mesh_handle.clone(),
+        transform: Transform::default()
```

Comment:
This could use `from_translation`.
Same for below :)

Reply:
Thanks for the hint. Changed it.